src/main/java/wolox/training/repositories/UserRepository.java

```diff
@@ -0,0 +1,8 @@
+package wolox.training.repositories;
+
+import org.springframework.data.repository.CrudRepository;
+import wolox.training.models.User;
+
+public interface UserRepository extends CrudRepository<User, Long> {
+    User findFirstByName(String name);
```

Comment:
Return a wrapped User (`Optional<User>`) rather than the User directly, in order to avoid nullchecks :)